apps/coscrad-frontend/src/components/resources/songs/song-detail.presenter.tsx

```diff
@@ -0,0 +1,6 @@
+import { IDetailQueryResult, ISongViewModel } from '@coscrad/api-interfaces';
+import { GenericDetailPresenter } from '../../../utils/generic-components/presenters/generic-detail-presenter';
+
+export const SongDetailPresenter = (
+    songAndActions: IDetailQueryResult<ISongViewModel>
+): JSX.Element => GenericDetailPresenter(songAndActions);
```

Comment:
JSX notation for the return would be more idiomatic, but I realize that this is just a temporary placeholder.